index.bs

```diff
@@ -2913,6 +2913,9 @@ The progress of navigation is communicated using an immutable
 
   <dt><dfn export for="WebDriver BiDi navigation status" id="navigation-status-url">url</dfn></dt>
   <dd>The URL which is being loaded in the navigation</dd>
+
+  <dt><dfn export for="WebDriver BiDi navigation status" id="navigation-status-suggested-filename">suggestedFilename</dfn></dt>
+  <dd>An optional suggested filename. Provided if the navigation is a download</dd>
```

Comment:
You can't have a struct field that can be unset; it needs to default to null, even though it's guaranteed to be non-null in the one case we access it.

Reply:
fixed